src/Symfony/Component/Cache/Adapter/TraceableAdapter.php

```diff
@@ -168,6 +168,19 @@ public function commit()
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function invalidateTags(array $tags)
+    {
+        $event = $this->start(__FUNCTION__, $tags);
+        try {
+            return $event->result = $this->pool->invalidateTags($tags);
```

Comment:
I still hesitate, calling `invalidateTags()` on it would gives the same error as when calling it on the inner directly, right?